lib/Sandbox.js

```diff
@@ -59,6 +59,14 @@ export type EnvironmentVarExport = {
   __BUILT_IN_DO_NOT_USE_OR_YOU_WILL_BE_PIPd?: boolean;
 };
 
+export type EsyConfig = {
+  build: ?string;
+  buildOutOfSource: boolean;
```

Comment:
Agreed, defaulting to `false` is more intuitive. Though `buildsInSource` name is not prone to typos — I'd write `buildInSource` most of the times. Which is by the way reminds me that we probably need `package.json` config validation in esy.